850.split-ambiguities/r.yaml

```diff
@@ -12,6 +12,9 @@
 - { name: rdd, wwwpart: r043v, setname: rdd-redis }
 - { name: rdd, warning: "please classify me" }
 
+- { name: redo, wwwpart: jdebp.eu, setname: redo-jdebp }
+- { name: redo, wwwpart: [redo.readthedocs.io, github.com/apenwarr], setname: redo-apenwarr }
+
```

Comment:
- Needs additional rules for repositories which lack homepage information, namely exherbo and solus.
- Also needs `- { name: redo, warning: "please classify me" }` to get notified when new packages which need split appear

Reply:
Any suggestion what I could use in the case of exherbo and solus? I'm not too familiar with the repology data structure.

Reply:
https://github.com/repology/repology-rules#ruleset

Reply:
Yes, thanks I found that. Is there any other place than https://repology.org/project/redo/information to see the actual metadata that is available for each distro/repo</

Reply:
https://repology.org/project/redo/packages has enough information for adding new rules

Reply:
Updated the PR to include solus and exherbo.